lib/utils.pm

```diff
@@ -239,6 +240,10 @@ sub is_jeos() {
     return get_var('FLAVOR', '') =~ /^JeOS/;
 }
 
+sub is_casp() {
```

Comment:
I expect to use it also in bootloader and maybe during installation, but I can move it to main.pm for now